extensions/agent/src/dialogs/proxyDialog.ts

```diff
@@ -23,28 +24,53 @@ export class ProxyDialog extends AgentDialog<ProxyData>  {
 	private static readonly ProxyNameTextBoxLabel: string = localize('createProxy.ProxyName', 'Proxy name');
 	private static readonly CredentialNameTextBoxLabel: string = localize('createProxy.CredentialName', 'Credential name');
 	private static readonly DescriptionTextBoxLabel: string = localize('createProxy.Description', 'Description');
-	private static readonly SubsystemsTableLabel: string = localize('createProxy.Subsystems', 'Subsystems');
-	private static readonly SubsystemNameColumnLabel: string = localize('createProxy.SubsystemName', 'Subsystem');
+	private static readonly SubsystemLabel: string = localize('createProxy.SubsystemName', 'Subsystem');
+	private static readonly OperatingSystemLabel: string = localize('createProxy.OperatingSystem', 'Operating system (CmdExec)');
+	private static readonly ReplicationSnapshotLabel: string = localize('createProxy.ReplicationSnapshot', 'Replication Snapshot');
+	private static readonly ReplicationTransactionLogLabel: string = localize('createProxy.ReplicationTransactionLog', 'Replication Transaction-Log Reader');
+	private static readonly ReplicationDistributorLabel: string = localize('createProxy.ReplicationDistributor', 'Replication Distributor');
+	private static readonly ReplicationMergeLabel: string = localize('createProxy.ReplicationMerge', 'Replication Merge');
+	private static readonly ReplicationQueueReaderLabel: string = localize('createProxy.ReplicationQueueReader', 'Replication Queue Reader');
+	private static readonly SSASQueryLabel: string = localize('createProxy.SSASQueryLabel', 'SQL Server Analysis Services Query');
+	private static readonly SSASCommandLabel: string = localize('createProxy.SSASCommandLabel', 'SQL Server Analysis Services Command');
+	private static readonly SSISPackageLabel: string = localize('createProxy.SSISPackage', 'SQL Server Integration Services Package');
+	private static readonly PowerShellLabel: string = localize('createProxy.PowerShell', 'PowerShell');
+
+
 
 	// UI Components
 	private generalTab: sqlops.window.modelviewdialog.DialogTab;
 
 	// General tab controls
 	private proxyNameTextBox: sqlops.InputBoxComponent;
-	private credentialNameTextBox: sqlops.InputBoxComponent;
+	private credentialNameDropDown: sqlops.DropDownComponent;
 	private descriptionTextBox: sqlops.InputBoxComponent;
-	private subsystemsTable: sqlops.TableComponent;
-
-	constructor(ownerUri: string, proxyInfo: sqlops.AgentProxyInfo = undefined) {
+	private subsystemCheckBox: sqlops.CheckBoxComponent;
+	private operatingSystemCheckBox: sqlops.CheckBoxComponent;
+	private replicationSnapshotCheckBox: sqlops.CheckBoxComponent;
+	private replicationTransactionLogCheckBox: sqlops.CheckBoxComponent;
+	private replicationDistributorCheckBox: sqlops.CheckBoxComponent;
+	private replicationMergeCheckbox: sqlops.CheckBoxComponent;
+	private replicationQueueReaderCheckbox: sqlops.CheckBoxComponent;
+	private SQLQueryCheckBox: sqlops.CheckBoxComponent;
```

Comment:
these variables should be camelCased, so `sqlQueryCheckBox`.  same for other `SQL` control variables.